extensions/amp-story/1.0/amp-story-access.css

```diff
@@ -82,10 +82,50 @@ amp-story-access.i-amphtml-story-access-visible::before {
   border-radius: initial !important;
 }
 
-/** access content. */
+/** Access header. */
+
+.i-amphtml-story-access-header {
+  position: relative !important;
+  height: 80px !important;
+  min-height: 80px !important;
+  background: var(--primary-color, #F0F0F0) !important;
+  z-index: 2 !important;
+
+  /* Making sure the background color fills the container in full bleed mode. */
+  margin: 0 -8px !important;
+}
+
+.i-amphtml-story-access-logo {
+  position: absolute !important;
+  bottom: -32px !important;
+  margin-left: -32px !important;
```

Comment:
did you test this with `dir=rtl`?

Reply:
I actually completely forgot RTL for this component... I'd also need to add the `dir` attribute on the shadow root. I'll do that in a follow up PR, thanks for the reminder!

Reply:
SG, thanks